tools/pycbc_test_suite.sh

```diff
@@ -58,9 +60,8 @@ if [ "$PYCBC_TEST_TYPE" = "search" ] || [ -z ${PYCBC_TEST_TYPE+x} ]; then
     fi
     popd
 
-
-    # run PyCBC Live test if running in Python 3
-    if [ "$PYTHON_VERSION" = "3" ]
+    # run PyCBC Live test if running in Python > 3.6
+    if [ "$PYTHON_VERSION" = "3" && "$PYTHON_MINOR_VERSION" -ge "7" ]
```

Comment:
If this starts to get nasty, you might consider a dedicated github action for the skymap testing (in the same way that the search workflow test is a dedicated action). That would be easier to set that for specific python versions then.

Reply:
Hmm, good point. It seems the current check makes the test pass, so I am inclined to go with it (and remove it if we explicitly require ≥3.7 at some point) rather than making a special action just for this little script.